src/ios/CodePush.m

```diff
@@ -314,6 +314,10 @@ - (void)saveFailedUpdate:(NSString *)packageHash {
     NSMutableArray* failedUpdates = [preferences objectForKey:FailedUpdatesKey];
```

Comment:
Cool. I modified the implementation to call `mutableCopy` right after we get the object. Hopefully it looks more clear now.